sdk/communication/azure-communication-sms/tests/test_sms_client.py

```diff
@@ -0,0 +1,63 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import unittest
+
+from azure.core.credentials import AccessToken
+from azure.communication.sms import SmsClient
+from unittest_helpers import mock_response
+
+try:
+    from unittest.mock import Mock
+except ImportError:  # python < 3.3
+    from mock import Mock  # type: ignore
+
+class FakeTokenCredential(object):
+    def __init__(self):
+        self.token = AccessToken("Fake Token", 0)
+
+    def get_token(self, *args):
+        return self.token
+
+class TestSMSClient(unittest.TestCase):
+    def test_send_message(self):
+        to_phone_number = "+18000005555"
+        raised = False
+
+        def mock_send(*_, **__):
+            return mock_response(status_code=202, json_payload={
+                "value": [
+                    {
+                        "to": to_phone_number,
+                        "messageId": "id",
+                        "httpStatusCode": "202",
+                        "errorMessage": "null",
+                        "repeatabilityResult": "accepted",
+                        "successful": "true"
+                    }
+                ]
+            })
+            
+        sms_client = SmsClient("https://endpoint", FakeTokenCredential(), transport=Mock(send=mock_send))
+
+        sms_response = None
+        try:
+            sms_responses = sms_client.send(
+                from_="+18000005554",
```

Comment:
nit: maybe we should use a constant test number like 18005551234 (this is the js one, it was defined in the recorder settings too when i took over).
that said, this isn't a dealbreaker